docs/import-data/files/cypherl.md

```diff
@@ -1,35 +1,62 @@
 ---
 id: cypherl
 title: Importing Cypher queries (.cypherl format)
-sidebar_label: Cypher queries (.cypherl format)
+sidebar_label: .cypherl
 pagination_prev: import-data/overview
 ---
 import Tabs from "@theme/Tabs"; import TabItem from "@theme/TabItem";
 
-When Memgraph is running, Cypher queries are imported by running
-[mgconsole](/connect-to-memgraph/mgconsole.md) in non-interactive mode and
-importing data saved in a `.cypherl` file. 
+If your data is in the form of Cypher queries (`CREATE` and `MERGE` clauses)
+within a **.cypherl** file it can be imported via Memgraph
+Lab or mgconsole.
 
-The great thing about importing data with the `.cypherl` file is that you need
-only one file to cover both nodes and relationships. But it can be tricky to
+The benefit of importing data using the `.cypherl` file is that you need only
+one file to import both nodes and relationships. But it can be tricky to
```

Comment:
Here I would add that for bigger files it can be slower, I didn't test this but @tonilastre mentioned that loading via CSV is much faster, I think running from CLI is the fastest. It is a thing we can mention somewhere, maybe not so relevant to this PR, it can be in different one.